recipe/meta.yaml

```
@@ -60,13 +62,13 @@ test:
     {% set tests_to_skip = tests_to_skip + " or test_to_datetime_format_YYYYMMDD_with_nat" %}   # [ppc64le]
     {% set tests_to_skip = tests_to_skip + " or (TestReductions and test_median_2d)" %}         # [ppc64le]
     {% set extra_args = extra_args + ["-k", "not (" + tests_to_skip + ")"] %}
-    - python -c "import pandas; pandas.test(extra_args={{ extra_args }})"  # [python_impl == "cpython"]
+    - python -c "import pandas, sys; sys.exit(pandas.test(extra_args={{ extra_args }}))"
   requires:
     - pip
     - pytest >=6.0
     - pytest-asyncio >=0.17
     - pytest-xdist >=1.31
```

I think this is needed to prevent running out of memory. I hope it doesn't cause other problems, since test collection works slightly differently.
```suggestion
    - pytest-xdist >=1.31  # [python_impl == "cpython"]
```